src/main/resources/db/migration/v0.30.0.1__linkml_v2.0.0_update.sql

```diff
@@ -0,0 +1,2574 @@
+UPDATE vocabularyterm t SET vocabulary_id = a.id FROM auditedobject a WHERE t.vocabulary_id = a.old_id AND a.tablename = 'vocabulary';
+UPDATE vocabularyterm_synonyms t SET vocabularyterm_id = a.id FROM auditedobject a WHERE t.vocabularyterm_id = a.old_id AND a.tablename = 'vocabularyterm';
+UPDATE vocabularytermset t SET id = a.id FROM auditedobject a WHERE t.id = a.old_id AND a.tablename = 'vocabularytermset';
+UPDATE vocabularytermset t SET vocabularytermsetvocabulary_id = a.id FROM auditedobject a WHERE t.vocabularytermsetvocabulary_id = a.old_id AND a.tablename = 'vocabulary';
+UPDATE vocabularytermset_vocabularyterm t SET vocabularytermsets_id = a.id FROM auditedobject a WHERE t.vocabularytermsets_id = a.old_id AND a.tablename = 'vocabularytermset';
+UPDATE vocabularytermset_vocabularyterm t SET memberterms_id = a.id FROM auditedobject a WHERE t.memberterms_id = a.old_id AND a.tablename = 'vocabularyterm';;
+UPDATE vtterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE wbbtterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE wblsterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE wbphenotypeterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE xbaterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE xbedterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE xbsterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE xcoterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE xpoterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE xsmoterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE zecoterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE zfaterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+UPDATE zfsterm t SET id = a.id FROM auditedobject a WHERE t.curie = a.old_curie AND a.tablename = 'ontologyterm';
+
+INSERT INTO submittedobject (id, modentityid, dataprovider_id) SELECT id, curie, dataprovider_id FROM biologicalentity;
+INSERT INTO submittedobject (id, modentityid, modinternalid, dataprovider_id) SELECT id, modentityid, modinternalid, dataprovider_id FROM reagent;
+
+INSERT INTO curieobject (id) SELECT id FROM submittedobject;
+INSERT INTO curieobject (id, curie) SELECT id, curie FROM ontologyterm;
+INSERT INTO curieobject (id, curie) SELECT id, curie FROM informationcontententity;
+
```

Comment:
I would guess you could remove the pkeys here or just let them drop when the column gets dropped.